core/services/offchainreporting/peer_wrapper.go

```diff
@@ -32,34 +32,26 @@ type (
 		PeerID        models.PeerID
 		Peer          peer
 
-		startMu *sync.Mutex
-		started bool
+		utils.StartStopOnce
 	}
 )
 
 // NewSingletonPeerWrapper creates a new peer based on the p2p keys in the keystore
 // It currently only supports one peerID/key
 // It should be fairly easy to modify it to support multiple peerIDs/keys using e.g. a map
 func NewSingletonPeerWrapper(keyStore *KeyStore, config *orm.Config, db *gorm.DB) *SingletonPeerWrapper {
-	return &SingletonPeerWrapper{keyStore, config, db, nil, "", nil, new(sync.Mutex), false}
+	return &SingletonPeerWrapper{keyStore, config, db, nil, "", nil, utils.StartStopOnce{}}
```

Comment:
ah rather than passing utils.StartStopOnce{} as value, maybe more readable to use named params for the first 3 fields, so you don't need to specify the others